asset-transfer-events/application-gateway-go/app.go

```diff
@@ -66,7 +66,7 @@ func main() {
 }
 
 func startChaincodeEventListening(ctx context.Context, network *client.Network) {
```

Comment:
Should this `app.go` file be in this PR?

Reply:
Yes, absolutely. The sample is not useful if it doesn't contain a sample application.

Reply:
You asked me to update the comments around stopping event listening